docs/external_contribution_request.md

```diff
@@ -0,0 +1,11 @@
+## Issue title
+
+Publish `EXTENSION_NAME` to Open VSX
+
+## Issue body
+
+Please consider publishing the extension to the OpenVSX registry.
+
+We are running the Web version of VS Code at Gitpod. The problem is, that only Microsoft products like VS Code may use the MS VS Code extensions Marketplace, so we must use Open VSX instead. It is a vendor neutral alternative. Also, products like VSCodium, Theia, code-server many other projects are using the registry to allow their users to install extensions. 
+
+The docs for publishing the extension are here: https://github.com/eclipse/openvsx/wiki/Publishing-Extensions. Technically this should be straightforward. You basically must get a token and call `ovsx publish` with it. For example, this is how RedHat publishes their java extension: redhat-developer/vscode-java@ff51dbf/Jenkinsfile#L73-L82
```

Comment:
This PR: https://github.com/open-vsx/publish-extensions/pull/483/files and this PR are closely related. 

I think we should reference the other PR docs from within this issue template (we can do in a subsequent PR, we don't need to couple them together). But it's so that we make it super clear in these docs to the extension maintainers _why_ it's important that they should publish the extensions themselves.

Reply:
Added a mention in [`87eee1b` (#479)](https://github.com/open-vsx/publish-extensions/pull/479/commits/87eee1b075673cc149db2ea33e0def14957827a4).